Integrations/integration-Wildfire.yml

```diff
@@ -254,10 +254,35 @@ script:
     description: Retrieve results for a file hash using WildFire
   - name: file
     description: Retrieve results for a file hash using WildFire
-    deprecated: true
```

Comment:
No results yet sounds like a good response to me as long as we're not throwing an error